lumibot/entities/order.py

```diff
@@ -533,7 +533,7 @@ def is_canceled(self):
         bool
             True if the order has been cancelled, False otherwise.
         """
-        return self.status.lower() in ['cancelled', 'canceled', 'cancel']
+        return self.status and self.status.lower() in ['cancelled', 'canceled', 'cancel']
```

Comment:
```self.status``` is always given a default value of "unprocessed" in the ```__init__()``` method.  What situation did you hit that made you decide you needed to include an existence check?  Seems like maybe there is a problem somewhere else that needs addressing.

Reply:
somehow i was getting a status of None, which was crashing the bot. not sure where it comes from, but definitely a possibilit